user_tools/setup.cfg

```diff
@@ -1,6 +1,2 @@
-[options.entry_points]
-console_scripts =
-    spark_rapids_user_tools = spark_rapids_pytools.wrapper:main
-    ascli = spark_rapids_pytools.ascli_cli.ascli:main
 [options.package_data]
```

Comment:
After this change, `setup.cfg` only contains entries for `[options.package_data]`. Should we move them to  `pyproject.toml` and remove this file?

Reply:
Thank @amahussein